contracts/upgradeable_contracts/GasTokenConnector.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity 0.4.24;
+
+import "../interfaces/IGasToken.sol";
+import "../upgradeable_contracts/Ownable.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+* @title GasTokenConnector
+* @dev This logic allows to use GasToken (https://gastoken.io)
+*/
+contract GasTokenConnector is Ownable {
+    using SafeMath for uint256;
```

Comment:
It looks like we are not using `SafeMath` methods here, I think we can remove the `import` and `using` statements

Reply:
Yes, you are right. I removed it